docs/404.html

```diff
@@ -32,7 +32,7 @@
       </button>
       <span class="navbar-brand">
         <a class="navbar-link" href="https://traitecoevo.github.io/austraits/index.html">austraits</a>
-        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.0.0</span>
+        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.1.1</span>
```

Comment:
Generally for docs, you wouldn't include this in the PR, as too many files to review. I'd commit docs in an extra commit after the PR is accepted. 

Also, you don't need to review changes to docs line-by-line, as we're never going to reconcile changes. Docs are always a wholesale update on. Though doesn't make sense to quickly scan changes to check there's nothing weird happened (e.g. mass deletion of content)

Reply:
Thanks for the tip @dfalster! I figured there was a more efficient way! I once had to fix the conflicts in each html file and it was painfully.

 I am going to git checkout the docs/ folder back to what it was on master branch and commit the docs folder on develop after merging the PR. 

Will follow this: https://blog.devgenius.io/how-to-remove-modified-or-changed-files-from-a-git-pull-request-a7769e3c58da